boards/arduino-mkr-common/Makefile.include

```diff
@@ -6,6 +6,10 @@ export CPU_MODEL = samd21g18a
 export PORT_LINUX ?= /dev/ttyACM0
 export PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 
+export ROM_OFFSET = 0x2000
```

Comment:
This should only be defined when using bossa programmer, with jlink programmer the bootloader is overwritten.

Reply:
Oh yes! I forgot, will change.